app/models/usergroup.rb

```diff
@@ -112,4 +116,12 @@ def ensure_last_admin_group_is_not_deleted
   def other_admins
     User.unscoped.only_admin.except_hidden - all_users
   end
+
+  def ssh_keys
+    users.flat_map(&:ssh_keys)
+  end
+
+  def ssh_authorized_keys
```

Comment:
Duplicate code (the Jail and this method) between User and Usergroup, please extract to a concern